build.gradle

```diff
@@ -212,15 +203,15 @@ subprojects {
   targetCompatibility = '1.8'
 
   dependencies {
-    implementation 'org.slf4j:slf4j-api'
+    implementation libs.slf4j.api
 
-    testImplementation 'org.junit.vintage:junit-vintage-engine'
-    testImplementation 'org.junit.jupiter:junit-jupiter-engine'
-    testImplementation 'org.junit.jupiter:junit-jupiter'
-    testImplementation 'org.slf4j:slf4j-simple'
-    testImplementation 'org.mockito:mockito-core'
-    testImplementation 'org.mockito:mockito-inline'
-    testImplementation 'org.assertj:assertj-core'
+    testImplementation libs.junit.vintage.engine
```

Comment:
Hint: JUnit engines should be referenced as `testRuntimeOnly`, same for the Jupiter engine below.

Reply:
done

Reply:
I had to revert it back as compileTestJava started to fail for iceberg-dell and iceberg-aliyun, If that's something that is not a part of a transition to VersionCatalog, can we do it separately?

Reply:
Just noting: even test code should not have a compile-time (*implementation *compileOnly etc) dependency to an engine. That's a project issue though.